File: fahrenheit_to_celsius.py
FREEZING_ZERO = 32

# Define the converter function
def fahrenheit_to_celsius(temp_f):
  return round((temp_f - FREEZING_ZERO) * 5/9, 2)

# Collect the user's input to be converted
user_query = int(float(input("Enter the temperature in Fahrenheit: ")))
print(f"You entered: {user_query} F")

# Convert the user's input and print the result
user_answer = str(fahrenheit_to_celsius(user_query))
print(f"{user_query} degrees Fahrenheit is {user_answer} degrees Celsius.")

File: fahrenheit_to_celsius.js
// requirement: make sure to `npm install readline-sync` before running the script

const FREEZING_ZERO = 32;

function fahrenheitToCelsius(tempFahrenheit) {
  return ((tempFahrenheit - FREEZING_ZERO) * 5 / 9);
}

let readlineSync = require('readline-sync');
let tempFahrenheit = readlineSync.question("Enter the temperature in Fahrenheit: ");
console.log(`You entered: ${tempFahrenheit} F`);

let convertedTemp = Math.round(fahrenheitToCelsius(tempFahrenheit));
console.log(`${tempFahrenheit} degrees Fahrenheit is ${convertedTemp} degrees Celsius.`)
